### PR TITLE
Add issues and PR templates

### PR DESCRIPTION
**Pull request by lholmes8**
Add template files to docs directory to help with issue and pull request submissions.
This pull request closes issue #35

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Before creating a pull request please make sure you have the following:
```
Comment (jonathonmcmurray) [inline PULL_REQUEST_TEMPLATE.md]: this file is in two places

**Comment by jonathonmcmurray** [comment]
Try previewing the markdown files on github, the headers are broken

Also interesting spelling of "description"...

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+##Description
+Enter a dexription of the pull request here
```
Comment (ThomasSmyth) [inline docs/PULL_REQUEST_TEMPLATE.md]: description

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+##Description
```
Comment (ThomasSmyth) [inline docs/PULL_REQUEST_TEMPLATE.md]: Need a space after #'s to ensure it displays correctly.

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -6,6 +10,7 @@ Before creating a pull request please make sure you have the following:
 - [ ] Colour-coded labels for categorisation.
 - [ ] A project to track PRs on a higher level
 
+##Optional Extras - remove if unused
```
Comment (ThomasSmyth) [inline docs/PULL_REQUEST_TEMPLATE.md]: Probably should be in it `_italics_` or as a `<!--comment-->` (which will be hidden from view).

File: docs/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -1,14 +1,19 @@
...
 - [ ] A description with a series of tasks in the form of a checklist
 - [ ] Assignees responsible for moving an issue forward
 - [ ] Colour-coded label for categorisation
 
+<!-- Optional Extras - remove if unused-->
 You may also need:
 
 - [ ] A milestone to associate an issue with specific development phases
 - [ ] A project to track issues and milestones on a higher level
 - [ ] Details of commits referencing the issue
```
Comment (ThomasSmyth) [inline docs/ISSUE_TEMPLATE.md]: Can remove this point, isn't related to the creation of the issue.

**Comment by ThomasSmyth** [comment]
Just noticed your commits for this aren't referencing the issue number (#35) and it isn't being closed in the description.

**Pull request merged**